src/controllers/product-controller.ts

```diff
@@ -86,3 +96,56 @@ export const checkoutRequest = async (ctx: Context) => {
             throw err;
     }
 };
+
+// Just for testing
```

Comment:
then move it to another file and dont commit it - unless you want to add full tests?

Reply:
removed it